dspace-server-webapp/src/main/java/org/dspace/app/rest/submit/step/DescribeStep.java

```diff
@@ -262,6 +260,10 @@ private Operation getOperationWithChangedMetadataField(Operation oldOp, String m
 
         // Operation has a value wrapped in the JsonValueEvaluator
         JsonValueEvaluator jsonValEvaluator = (JsonValueEvaluator) oldOp.getValue();
+        if (jsonValEvaluator == null) {
+            log.warn("The complex input field is not processed, because the operation doesn't have a value.");
```

Comment:
```suggestion
            log.warn("The complex input field is not processed, because the operation is null.");
```
but what kind of operation?

Reply:
That is an instance of the `Operation` class. It could be `Add/Remove/Edi/..?t` operation.